scrape/manager.go

```diff
@@ -130,6 +134,20 @@ func (m *Manager) reload() {
 	wg.Wait()
 }
 
+// setServerJitter calculates global jitter per server relying on extra label set.
+func (m *Manager) setServerJitter(labels model.LabelSet) error {
+	h := fnv.New64a()
+	hostname, err := os.Hostname()
+	if err != nil {
+		return err
+	}
+	if _, err := h.Write([]byte(hostname + labels.String())); err != nil {
+		return err
+	}
+	m.serverJitter = time.Duration(h.Sum64()%60) * time.Second
```

Comment:
You've still got the 60s in here.

Reply:
that's a trade-off between distribution and collisions. 
It could affect scrape intervals more than minute, but it's what target hash does in here: https://github.com/prometheus/prometheus/pull/5181/files#diff-71e6b8df5f2eeb6a786cb5cacaa4187dR135 it appends jitter up to scrape interval.

Reply:
There's no need for it though.

Reply:
you mean modulo operation?

Initially I wanted to pass to the `scrape` interface an offset as a time.Duration, that's why I called it a jitter and implemented as a time offset
https://github.com/prometheus/prometheus/pull/5181/files#diff-6a976999031a4966ff9d1a2734f15e5dR439

Otherwise passing a hash seed would be vague and cause questions like: why is this hash in here.

I might should have add more clear comment, that jitter in interval [0-59) will not affect postpone scrapes on more than the interval time itself.

But I hope with a current implementation: jitter speaks for itself what is this for.

Reply:
Passing the hash as an int would be better, it'll keep all the math in one place.